pinatapy/__init__.py

```diff
@@ -49,9 +67,9 @@ def get_all_files(directory: str) -> tp.List[str]:
 
         if os.path.isdir(path_to_file):
             all_files: tp.List[str] = get_all_files(path_to_file)
-            files = [("file",(file, open(file, "rb"))) for file in all_files]
+            files = [("file", (dest_folder_name + file.split("/")[-1], open(file, "rb"))) for file in all_files]
```

Comment:
I experience some issues testing the script rn due to Pinata services down, but won't this work incorrect with several sub-folders?
For a structure like 
```
parent_folder
| subfolder_1
|| file_1
| subfolder_2
|| file_2
```
and `ipfs_destination_path` = `parent_folder` won't the ipfs folder structure be like
```
parent_folder
| file_1
| file_2
```
since you take `file.split("/")[-1]` which is just a filename without path?

Reply:
The functionality only allows you to set one destination folder. So all items in that folder will be nested inside the `ipfs_destination_path` folder you enter.